helm/alfresco-content-services/tests/deployment-repository_test.yaml

```diff
@@ -195,3 +195,59 @@ tests:
               cpu: "4"
               memory: "3000Mi"
         template: deployment-repository.yaml
+
+  - it: should have wait-db-ready initcontainer with external database
+    values: *testvalues
+    set:
+      database.external: false
+    asserts:
+      - equal:
+          path: spec.template.spec.initContainers[0].name
+          value: wait-db-ready
+        template: deployment-repository.yaml
+
+  - it: should not have wait-db-ready initcontainer with external database
+    values: *testvalues
+    set:
+      database.external: true
+    asserts:
+      - isEmpty:
```

Comment:
This test will break if we add an initContainer later on. Can't [`notContains`](https://github.com/helm-unittest/helm-unittest/blob/main/DOCUMENT.md#assertion-types) fulfill this check?

Reply:
`contains` assertions works great with simple objects, but in this case we have a complex object and in order to pass I have to provide the full specs that in case of an initcontainer are pretty big (e.g. `command`) and I can't provide a subset of them (e.g. just check the name of the container).

I decided to go with isEmpty but indeed I am not satisfied with it